riff-raff/app/views/deploy/form.scala.html

```diff
@@ -75,9 +75,11 @@
                 helper.options(prismLookup.stages.toList),
                 Symbol("_default") -> "--- Choose a stage ---",
                 Symbol("_label") -> "Stage",
+                Symbol("_help") -> "Nb. you can now constrain available stages by setting 'allowedStages' in your riff-raff.yaml.",
```

Comment:
I think we should move to making this a preferred setting, and as such include it as a deployment warning (for example, the [naming of the default branch](https://github.com/guardian/riff-raff/pull/599)).

This might be for follow-up PR though after we test it out on some projects.

Reply:
Great shout and yep, agree it is sensible to do as a follow-on.